src/tools/check_brokenURL.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+
+import re
+import os
+import sys
+import getopt
+import requests
+import argparse
+
+def get_filepaths(directory):
+    file_paths = []  
+    for root, directories, files in os.walk(directory):
+        for filename in files:
+            filepath = os.path.join(root, filename)
+            file_paths.append(filepath)
+    return file_paths
+
+def checkBrokenLink(url):
+    if requests.get(url).ok == True:
```

Comment:
This is more of a managerial decision, I believe if it is being redirected, then it should still be a broken link.
It should be okay only when there is HTTP 200.
I am new to Ceph and final call should be taken by you ( @Liuchang0812  ) and other senior developers.